dspace-spring-rest/src/test/java/org/dspace/app/rest/RelationshipRestRepositoryIT.java

```diff
@@ -0,0 +1,2118 @@
+            }
+        }
+
+
+        list = itemService.getMetadata(publication, "dc", "contributor", "author", Item.ANY);
+
+        for (MetadataValue mdv : list) {
+            if (StringUtils.equals(mdv.getValue(), "plain text three")) {
+                assertEquals(4, mdv.getPlace());
+            }
+        }
+
+
+        getClient(adminToken).perform(get("/api/core/relationships/" + thirdRelationshipIdString))
+                             .andExpect(status().isOk())
+                             .andExpect(jsonPath("leftPlace", is(3)));
+
+    }
+
+    /**
+     * This test will simply add Relationships between Items with a useForPlace attribute set to false for the
+     * RelationshipType. We want to test that the Relationships that are created will still have their place
+     * attributes handled in a correct way
+     * @throws Exception
+     */
+    @Test
+    public void addRelationshipsNotUseForPlace() throws Exception {
+
+        context.turnOffAuthorisationSystem();
+
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                                          .withName("Parent Community")
+                                          .build();
+        Community child1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
+                                           .withName("Sub Community")
+                                           .build();
+        Collection col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1").build();
+        Collection col2 = CollectionBuilder.createCollection(context, child1).withName("Collection 2").build();
+        Collection col3 = CollectionBuilder.createCollection(context, child1).withName("OrgUnits").build();
+
+        Item author1 = ItemBuilder.createItem(context, col1)
+                                  .withTitle("Author1")
+                                  .withIssueDate("2017-10-17")
+                                  .withAuthor("Smith, Donald")
+                                  .withRelationshipType("Person")
+                                  .build();
+
+        Item author2 = ItemBuilder.createItem(context, col2)
+                                  .withTitle("Author2")
+                                  .withIssueDate("2016-02-13")
+                                  .withAuthor("Smith, Maria")
+                                  .withRelationshipType("Person")
+                                  .build();
+
+        Item author3 = ItemBuilder.createItem(context, col2)
+                                  .withTitle("Author3")
+                                  .withIssueDate("2016-02-13")
+                                  .withAuthor("Maybe, Maybe")
+                                  .withRelationshipType("Person")
+                                  .build();
+
+        Item orgUnit1 = ItemBuilder.createItem(context, col3)
+                                   .withTitle("OrgUnit1")
+                                
```

Comment:
@abollini : Github seems to display comments in different orders based on whether you are looking at the PR homepage or the code source. When you reference other comments, could you please either link them, or copy the entire comment in?  It's difficult to understand what "previous comment" you are referring to, and I don't want to make assumptions.

Reply:
it is strange. If I look to the conversation tab I see it in the wrong place but it looks fine now in the files tab (so yes, it is a github bug).
The comment is about 
https://github.com/DSpace/DSpace/pull/2376/files#diff-61a3266941bcb44b91094e4d3f330ba6R1172
```
 Map<String, Object> map = mapper.readValue(content, Map.class);
        String firstRelationshipIdString = String.valueOf(map.get("id"));
```
and I'm point to my previous comment here https://github.com/DSpace/DSpace/pull/2376/files#r279211896

Reply:
@abollini : While I understand the desire to want to modify all the Tests to use that `.andDo` syntax (in this comment and in several of the others below), I think we should refrain from holding up this PR for minor syntax changes.  As-is, this IT test is written in an acceptable way...but, yes, there are more simplistic ways of achieving the same result.

Reply:
I would appreciate a note or a low priority ticket to go back on that. the `.andDo` syntax is not only more "simplistic" but also incentive to write better cleanup code in case of failure as additional resources created via rest without the use of builder are immediately identified.